src/mmrelay/main.py

```
@@ -421,6 +488,14 @@ def signal_handler() -> None:
     except KeyboardInterrupt:
         shutdown()
     finally:
+        if node_name_refresh_task is not None:
+            node_name_refresh_task.cancel()
+            try:
+                await node_name_refresh_task
+            except asyncio.CancelledError:
+                pass
+            except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)